reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -356,8 +356,16 @@ class StaticHtmlReporter : TableReporter() {
                             <td>${entry.scopes.joinToString(separator = "<br/>")}</td>
                             <td>${entry.declaredLicenses.joinToString(separator = "<br/>")}</td>
                             <td>${entry.detectedLicenses.joinToString(separator = "<br/>")}</td>
-                            <td>${entry.analyzerErrors.joinToString(separator = "<br/>")}</td>
-                            <td>${entry.scanErrors.joinToString(separator = "<br/>")}</td>
+                            <td><ul>
+                                ${entry.analyzerErrors.joinToString(separator = "<br/>") {
```

Comment:
The `separator` now separates `<li>` list items. So does `<br/>` still make sense to use? Maybe just use `separator = ""` or `separator = "\n"` instead? Similar below.

Reply:
Done.